docs/specification/1.0.md

```diff
@@ -139,12 +139,11 @@ Note: the `hookInstance` is globally unique and should contain enough entropy to
 
 #### prefetch
 
-As a performance tweak, the EHR may pass along data according
+As a performance tweak, the EHR MAY pass along data according
 to the service's [[Prefetch-Template]]. This helps provide the service with all
 the data it needs to efficiently compute a set of recommendations. Each key matches a key described in the CDS Service Discovery document; and each value is a FHIR Bundle.entry indicating a response status and returned resource.
 
-Note that in the absence of `prefetch`, an external service can always execute
-FHIR REST API calls against the EHR server to obtain additional data ad-hoc.)
+Note that in the absence of `prefetch`, a CDS service MAY present the access token passed by the EHR in the `fhirAuthorization` parameter to retrieve additional FHIR resources from the EHR FHIR server.   
```

Comment:
Use of the "fhirAuthorization" token in this context is incidental so it seems like a distraction to mention it here. Suggest retaining the original working, or something like:

Note that if `prefetch` data is not provided as part of the request, the service can always execute FHIR REST API calls against the provided FHIR URL to obtain additional data to process the request.

Reply:
I think the question here is around the opening line to this sentence which may give the impression that `fhirAuthorization` is only used if the prefetch request is not honored. Perhaps we simply reword this to:

"In addition to requesting data through `prefetch`, a CDS service..."

Reply:
I changed the "Note that..." to Kevin's suggested wording.  
However, I don't agree that a service "can always..." -- a service can retrieve data through the FHIR API only if the EHR has provided an access token.  I have reworded accordingly.